examples/notebooks/b - Secure Classification with TFE Keras - Secure Model Serving .ipynb

```diff
@@ -0,0 +1,302 @@
+{
```

Comment:
<p>the first `global request_ix` can be removed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tf-encrypted/tf-encrypted/pull/571/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>